### ISSUE TITLE
query about hyper parameters ?

### ISSUE THREAD

**Issue by silverlining21** [ppwwyyxx/tensorpack#71]
Hi, 
I train my network, which is reduce half kernels from **Alexnet**, on my own train set (about 100k images).
the hyper parameters set follow the example of **alexnet-dorefa.py** provided by you.

but I found something confused. in the example you init the learning rate to `1e-4` for train the model from scratch? but as far as I know in **caffe** we usually use a `higher learning rate` combine with momentum or step decay for trainning. the learning rate `1e-4` more likely to use  in finetune  a well trained model.

finally, do you have any suggestions or experience on how to set hyper parameter in tensorflow , such as `batcsize` / `optimizer` / `learning rate `? Any advices will be  sincerely appreciated.

**Comment by ppwwyyxx** [comment]
1e-4 or 1e-3 is usually good for Adam optimizer.

batchsize/optimizer/learning rate are all in the example. You can just change them.

**Comment by silverlining21** [comment]
OK, thanks. I will try out.
